core/frontend/src/extension/Extension.ts

```diff
@@ -9,9 +9,7 @@
 /**
  * @alpha
  */
-export enum ActivationEvent {
-  onStartup = "onStartup",
-}
+export type ActivationEvent = "onStartup";
```

Comment:
Is the thought with this change that it will be a union type instead of an Enum of strings?

Reply:
Yes, mentioned [here as well](https://github.com/iTwin/itwinjs-core/pull/3347#discussion_r833407873). Enum was also causing some circular dependency issues since it has to be imported as a value I believe

Reply:
Ah okay, sounds good.